ci/long_running_tests/config.yaml

```diff
@@ -64,7 +51,11 @@ head_setup_commands: []
 worker_setup_commands: []
 
 # Command to start ray on the head node. You don't need to change this.
-head_start_ray_commands: []
+head_start_ray_commands:
+    - source activate tensorflow_p36 && ray stop
+    - source activate tensorflow_p36 && ulimit -n 65536; ray start --head --redis-port=6379 --object-manager-port=8076 --autoscaling-config=~/ray_bootstrap_config.yaml
```

Comment:
Currently we are leaving it up to each stress test to start the Ray cluster so that they can choose to simulate a multinode cluster. I think it's probably simpler to leave it that way.